modules/lang-mustache/src/main/java/org/elasticsearch/script/mustache/TransportSearchTemplateAction.java

```diff
@@ -85,13 +86,13 @@ public void onFailure(Exception t) {
             } else {
                 listener.onResponse(response);
             }
-        } catch (IOException e) {
+        } catch (Exception e) {
             listener.onFailure(e);
         }
     }
 
     static SearchRequest convert(SearchTemplateRequest searchTemplateRequest, SearchTemplateResponse response, ScriptService scriptService,
-                                 NamedXContentRegistry xContentRegistry) throws IOException {
+                                 NamedXContentRegistry xContentRegistry) throws Exception {
```

Comment:
Is this needed ? We throw a runtime exception so it should be ok to leave the `IOException` as is.